bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -258,61 +263,58 @@ fn main() -> Result<()> {
                         Event::Metric(event) => {
                             debug!("Metric event: {:?}", event);
                         }
-                        Event::Telemetry(event) => {
-                            logs_agent.send_event(event.clone());
```

Comment:
I'm having a bit of trouble reading this diff, the only difference is you removed this send_event line, and it changed the identation of the rest of the code right?

Reply:
Correct, it's not being sent anymore, since we changed the place where it's being sent